### PR TITLE
adding the image list

### PR DESCRIPTION
**Pull request by AlvaroDazaQuiroga**

### CODE REVIEW

File: Imago/src/ui/ImageListModel.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+     * this method is in charge of filling the list model which ImageIcons
+     * which will return to the LeftPanel class for her to be loaded
+     *
+     * @return returns the  variable  listModel filled whit imagesIcon
+     */
+    public DefaultListModel listModel() {
+        try {
+            int count = 0;
+            for (int i = 0; i < listOfFiles.length; i++) {
+                System.out.println("check path" + listOfFiles[i]);
```
Comment (AlvaroDazaQuiroga) [inline Imago/src/ui/ImageListModel.java:31]: remove these println

File: Imago/src/ui/ImageListModel.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                String name = listOfFiles[i].toString();
+                if (name.endsWith("jpg")) {
+                    ImageIcon imageIcon = new ImageIcon(ImageIO.read(listOfFiles[i]));
+                    Image image = imageIcon.getImage();
+                    Image neWing = image.getScaledInstance(100, 100, Image.SCALE_SMOOTH);
+                    ImageIcon image2 = new ImageIcon(neWing);
```
Comment (AlvaroDazaQuiroga) [inline Imago/src/ui/ImageListModel.java:37]: Remove this one and use the imageIcon variable

File: Imago/src/ui/ImagoMainWindow.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+    private JMenuItem JMIOpen;
+    private JMenuItem JMISave;
+    private JMenuItem JMIExit;
+
+    /**
+     * This is the class constructor charging methods instance and add the components
+     * recive the "title" of the frame of the Main class
+     *
+     * @param title
```
Comment (AlvaroDazaQuiroga) [inline Imago/src/ui/ImagoMainWindow.java:28]: documentation for the parameter

File: Imago/src/ui/ImagoMainWindow.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        super(title);
+        initComponents();
+        addComponents();
+
+    }
+
+    /**
+     * This method instance the UI objects
+     */
+
```
Comment (AlvaroDazaQuiroga) [inline Imago/src/ui/ImagoMainWindow.java:40]: remove the empty line

File: Imago/src/ui/LeftSearchPanel.java [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+        toolbar = new Toolbar();
+        imageListModel = new ImageListModel();
+
+        imagesList = new JList(imageListModel.listModel());
+        imagesList.setVisibleRowCount(-1);
+        imagesList.setLayoutOrientation(JList.HORIZONTAL_WRAP);
+        imagesList.setSelectionMode(ListSelectionModel.MULTIPLE_INTERVAL_SELECTION);
+        setLayout(new BorderLayout());
+        new JPanel();
```
Comment (AlvaroDazaQuiroga) [inline Imago/src/ui/LeftSearchPanel.java:42]: remove this line

File: Imago/src/ui/Toolbar.java [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+     * @param Path
+     */
+    public void fileChooser(String Path) {
+        JFileChooser fileChooser = new JFileChooser();
+        fileChooser.setFileSelectionMode(JFileChooser.DIRECTORIES_ONLY);
+        fileChooser.setCurrentDirectory(new java.io.File("."));
+        int returnValue = fileChooser.showOpenDialog(null);
+        if (returnValue == JFileChooser.APPROVE_OPTION) {
+            File selectedFile = fileChooser.getSelectedFile();
+            System.out.println(selectedFile.getAbsolutePath());
```
Comment (AlvaroDazaQuiroga) [inline Imago/src/ui/Toolbar.java:60]: remove the println

File: Imago/src/ui/RightPanelViewer.java [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+/**
+ *IN CONSTRUCTION
+ * this clas will be in charge for managing the tools for modifying
+ * images such as resize, rotate ....
+ *
+ */
+public class RightPanelViewer extends JPanel {
+
+    private ImageIcon imageIcon;
+    private JLabel jImage;
```
Comment (AlvaroDazaQuiroga) [inline Imago/src/ui/RightPanelViewer.java:14]: use meaningful names for the variables and attributes

**Comment by edson-gonzales** [comment]
correct these observations and then we will review it again

**Pull request closed without merging**